narwhals/_spark_like/expr.py

```diff
@@ -436,13 +447,9 @@ def _is_finite(_input: Column) -> Column:
 
     def is_in(self: Self, values: Sequence[Any]) -> Self:
         def _is_in(_input: Column) -> Column:
-            return _input.isin(values)
+            return _input.isin(values) if values else self._F.lit(False)  # noqa: FBT003
```

Comment:
Related to #2031

Reply:
does it work to use `contains`, as per the latest change?

Reply:
I might have missed a lot this week. Not sure which change you refer to 🤔

Reply:
Ok I might be missing something - pyspark contains has different behaviour, and for sqlframe I rather not specialize for a specific dialect

Reply:
ah ok if pyspark's contains is different let's keep it like this 👍

Reply:
Just to expand a bit on this: [pyspark.sql.functions.contains](https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.contains.html#pyspark-sql-functions-contains) would be the equivalent of `.str.contains`